tasks/task772_pawsx_french_text_modification.json

```diff
@@ -0,0 +1,1547 @@
+{
+    "Contributors": [
+        "Jacob Anderson"
+    ],
+    "Source": [
+        "paws-x",
+        "https://huggingface.co/datasets/paws-x"
+    ],
+    "Categories": [
+        "Text Modification"
+    ],
+    "Definition": [
+        "Given a sentence in French, provide an equivalent paraphrased version from the original that retains the same meaning."
+    ],
```

Comment:
Could you add language tags for each of these task files?  #113

Reply:
Yes, I have added the `Input_language` and `Output_language` fields to the text modification tasks (770-776). I used the two-digit ISO 639-1 codes as suggested in the #113 for each respective language.

Additionally, since my previous translation instruction tasks used non-ISO codes, I have updated those as well.